packages/browser-destinations/src/destinations/segment-utility-web/throttle/index.ts

```diff
@@ -0,0 +1,50 @@
+import { SegmentUtilityInstance } from '..'
+import type { BrowserActionDefinition } from '../../../lib/browser-destinations'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+const action: BrowserActionDefinition<Settings, SegmentUtilityInstance, Payload> = {
+  title: 'Throttle',
+  description: 'Throttle events sent to Segment.',
```

Comment:
```suggestion
  description: 'Throttle events sent to Segment. Throttling is on a per-event name basis.',
```

I think something to distinguish that the throttling is not specific to this subscription, but per-events for all events that match this subscription.